src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+@import './utils/variables';
+@import './blocks/stars';
+@import './blocks/card';
+
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  line-height: 18px;
```

Comment:
Move font styles to the html selector

Reply:
Sorry but I I can't understand what these styles are in your screen, project deployment provides styles from the previous task, with first 4 active stars.
Agree with styles should be moved to html selector, and even AI-mentor told me this, but could you show the rule in our code styles rules that I can be sure further

Reply:
Not sure such a rule exists in `code styles rules` it's not about code style (at least not only). I guess it was mentioned in the theory and video about fonts. And it's a general rule - font-family (and similar to font-weight) should be the same for the entire page (with some exceptions, in these cases it overrides in places it is needed) and it will be inherited. So there is no need to write it for each tag with text